src/Files.App/UserControls/SidebarControl.xaml.cs

```diff
@@ -190,18 +187,18 @@ private void NotifyPropertyChanged([CallerMemberName] string propertyName = "")
 
 		private List<ContextMenuFlyoutItemViewModel> GetLocationItemMenuItems(INavigationControlItem item, CommandBarFlyout menu)
 		{
-			ContextMenuOptions options = item.MenuOptions;
+			var options = item.MenuOptions;
 
 			var favoriteModel = App.SidebarPinnedController.Model;
-			int favoriteIndex = favoriteModel.IndexOfItem(item);
-			int favoriteCount = favoriteModel.FavoriteItems.Count;
+			var favoriteIndex = favoriteModel.IndexOfItem(item);
+			var favoriteCount = favoriteModel.FavoriteItems.Count;
 
-			bool isFavoriteItem = item.Section is SectionType.Favorites && favoriteIndex is not -1;
-			bool showMoveItemUp = isFavoriteItem && favoriteIndex > 0;
-			bool showMoveItemDown = isFavoriteItem && favoriteIndex < favoriteCount - 1;
+			var isFavoriteItem = item.Section is SectionType.Favorites && favoriteIndex is not -1;
+			var showMoveItemUp = isFavoriteItem && favoriteIndex > 0;
+			var showMoveItemDown = isFavoriteItem && favoriteIndex < favoriteCount - 1;
 
-			bool isDriveItem = item is DriveItem;
-			bool isDriveItemPinned = isDriveItem && (item as DriveItem).IsPinned;
+			var isDriveItem = item is DriveItem;
+			var isDriveItemPinned = isDriveItem && (item as DriveItem).IsPinned;
```

Comment:
I argue that changing simple types to `var` is not preferred because it abstracts away the type which makes the code less clear for the future.

Reply:
While I would agree here, we're usually using `var` for variable declarations when we can in our codebase. It's more for the sake of consistency here.

Reply:
Well, if I open any file, any variable is most likely gonna be instanciated using `var` rather than an explicit time.

Reply:
That's true but isn't it bad practice?

Reply:
It really depends in my eyes. Basically, when the type of the variable inferred easily by the develop, it's repetition. 

Like in the case of `List<SomeVeryLongNameForArgumentSake> l = new List<SomeVeryLongNameForArgumentSake>` is very repetitive and dull. With a simple `var`, we cut the size in half and retains the same information.

In other case, when the type of return is not sure, then maybe it could be worth to write explicitly the type. But in any case, with the modern IDEs, the information can be found simply but hovering `var`.

In the end, it depends how much we want to apply the 'Don't Repeat Yourself' principle.

Reply:
This change is fine with me, you can hover over the variable to see the type.